### PR TITLE
added EOSC-related orgs to curators

### PR DESCRIPTION
**Pull request by allysonlister**
related to #1957

### CODE REVIEW

**Review by ramGranell — APPROVED** [review]

File: src/data/communityCurationCohorts.json [hunk 1]
```diff
@@ -192,6 +192,11 @@
             "id": 1974,
             "name": "CNRS",
             "tooltip": "The CNRS (National Center for Scientific Research)"
+          },
+          {
+            "id": 3831,
+            "name": "SSHOC",
+            "tooltip": "SSHOC"
```
Comment (prakhyatox) [inline src/data/communityCurationCohorts.json]: Tooltip should be `Social Sciences and Humanities Open Cloud`

File: src/data/communityCurationCohorts.json [hunk 2]
```diff
@@ -304,6 +314,11 @@
             "id": 2816,
             "name": "OU",
             "tooltip": "The Open University"
+          },
+          {
+            "id": 3829,
+            "name": "ESCAPE",
+            "tooltip": "ESCAPE"
```
Comment (prakhyatox) [inline src/data/communityCurationCohorts.json]: Tooltip should be `European Science Cluster of Astronomy & Particle physics ESFRI research infrastructures`

Comment (allysonlister) [reply src/data/communityCurationCohorts.json]: I've updated this now, thanks!

File: src/data/communityCurationCohorts.json (hunk 1 above)
Comment (allysonlister) [reply src/data/communityCurationCohorts.json]: I've updated this now, thanks!

**Review by prakhyatox — APPROVED** [review]

**Comment by allysonlister** [comment]
Thanks - who does the actual progression of this ticket now that the reviewers have approved? Should I, or should someone else - I don't do this enough to know the next steps :-)

**Comment by ramGranell** [comment]
> Thanks - who does the actual progression of this ticket now that the reviewers have approved? Should I, or should someone else - I don't do this enough to know the next steps :-)

You can merge it to Dev, with the button on the bottom that said "Merge pull request". Then it will be in dev and tomorrow may be merged to master and deployed in production.

**Comment by prakhyatox** [comment]
1) Once the reviewers have approved, click ```Merge pull request``` button (which is at the bottom in blue color) and it will merge the PR/code to dev branch. 
2) Check the output on https://preview.fairsharing.org/.
3) Milo does the merge ```dev``` branch to ```master``` and then it is on prod site

**Pull request merged**